cobigen-maven/cobigen-maven-systemtest/src/test/resources/testdata/systemtest/Devon4JTemplateTest/templates-nomatch/pom.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>templates-oasp4j</artifactId>
```

Comment:
Why inside the `Devon4JTemplateTest` do we have an artifactId `templates-oasp4j`?

Reply:
Happened because I've copied it directly from the Oasp4j tests... missed to check it I guess. It's adjusted now.